packages/sails-ng-common/src/validation/form.model.ts

```diff
@@ -37,14 +43,46 @@ export type FormValidatorControl = {
    * @param errors The complete map of validation errors.
    */
   setErrors(errors: FormValidatorErrors | null): void;
-};
+}
+
+/**
+ * A simple form validator control that can be used on the server-side to mimic client-side controls.
+ *
+ * Do not use this on the client-side.
+ */
+export class SimpleServerFormValidatorControl implements FormValidatorControl {
+    value: unknown;
+    private _setErrors: (FormValidatorErrors | null)[] = [];
+
+    constructor(value: unknown) {
+        this.value = value;
+    }
+
+    get<P extends string>(path: P): FormValidatorControl | null {
+        const result = _get(this.value, path) ?? null;
+        console.debug(`SimpleServerFormValidatorControl.get path '${path}' with result '${JSON.stringify(result)}' from value '${JSON.stringify(this.value)}'`);
```

Comment:
There's probably other instances of this but we should use the logger service instead of console.x so we can easily turn them off in prod mode